### PR TITLE
Fixed Cassandra icon style name

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

File: app/styles/_openshift-logos-icon.less [hunk 1]
```diff
@@ -191,6 +191,6 @@
 .icon-apache:before {
   content: "\f138";
 }
-.icon-casadra:before {
+.icon-casadra:before, .icon-cassandra:before {
```
Comment (spadgett) [inline app/styles/_openshift-logos-icon.less]: Might be worth a comment here not to remove `icon-casadra` for backwards compatibility.

**Comment by sg00dwin** [comment]
@spadgett I believe it would be fine

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
Merging since @sg00dwin made the other suggested change in #1496

**Pull request merged**